jina/peapods/zmq/__init__.py

```diff
@@ -495,7 +495,11 @@ def send_message(
     num_bytes = 0
     try:
         _prep_send_socket(sock, timeout)
-        sock.send_multipart(msg.dump())
+        if 'flags' in kwargs:
```

Comment:
we still have no reason for this right?

Reply:
@florian-hoenicke said there were some issues, when not setting it. I can remove it and we see.